### PR TITLE
Generate a filter named Visual Content.

### PR DESCRIPTION
**Pull request by KevinVillanuevaCordero**
Create a class FilterVisualContent in the module, that enables user to select the filter when create o modifies a textFormat.

### CODE REVIEW

File: src/Plugin/Filter/FilterVisualContent.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (danielnv18) [inline src/Plugin/Filter/FilterVisualContent.php]: @KevinVillanuevaCordero Please removed this comments and replace it with the comment @file

File: src/Plugin/Filter/FilterVisualContent.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function tips($long = FALSE) {
+
+
+  }
+
+}
```
Comment (danielnv18) [inline src/Plugin/Filter/FilterVisualContent.php]: @KevinVillanuevaCordero Always leave a empty space at the end of each php file

File: src/Plugin/Filter/FilterVisualContent.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+use Drupal\filter\Plugin\FilterBase;
+
+
+/**
+ * Provides a filter to use shortdoces.
+ *.
+ *
+ * @Filter(
+ *   id = "filter_visualcontent",
+ *   title = @Translation("Filtrate Visual Content"),
```
Comment (alarez) [inline src/Plugin/Filter/FilterVisualContent.php]: Change this line to read 
-   title = @Translation("Visual Content Layout"),

File: src/Plugin/Filter/FilterVisualContent.php [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+ *   description = @Translation("Use shortcodes to generate a easy content."),
```
Comment (alarez) [inline src/Plugin/Filter/FilterVisualContent.php]: Change this line to read 
-   description = @Translation("Provides a ShortCode filter format to easily generate content layout."),

File: src/Plugin/Filter/FilterVisualContent.php [hunk 5]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @file
+ * Filter that enables to use patterns for create visual content more easy.
```
Comment (alarez) [inline src/Plugin/Filter/FilterVisualContent.php]: - Filter that enable the use patterns to create visual content more easy.

**Comment by danielnv18** [comment]
@KevinVillanuevaCordero check @alarez comments in the code

**Pull request merged**